tests/contrib/timeseries/test_lgssm.py

```diff
@@ -1,45 +1,62 @@
 import torch
 
 from tests.common import assert_equal
-from pyro.contrib.timeseries import GenericLGSSM
+from pyro.contrib.timeseries import GenericLGSSM, GenericLGSSMWithGPNoiseModel
 import pytest
 
 
+@pytest.mark.parametrize('model_class', ['lgssm', 'lgssmgp'])
 @pytest.mark.parametrize('state_dim', [2, 3])
 @pytest.mark.parametrize('obs_dim', [2, 4])
 @pytest.mark.parametrize('T', [11, 17])
-def test_generic_lgssm_forecast(state_dim, obs_dim, T):
+def test_generic_lgssm_forecast(model_class, state_dim, obs_dim, T):
```

Comment:
we can reuse this test to test the trickiest part of the lgssmgp forecast method